nucypher/cli/commands/contacts.py

```diff
@@ -0,0 +1,116 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import shutil
+
+import click
+import os
+
+from nucypher.characters.control.emitters import StdoutEmitter
+from nucypher.cli.actions.select import select_card
+from nucypher.cli.options import option_force
+from nucypher.cli.painting.policies import paint_single_card, paint_cards
+from nucypher.cli.types import EXISTING_READABLE_FILE
+from nucypher.policy.identity import Card
+
+
+@click.group()
+def contacts():
+    """"Manage character cards"""
+
+
+@contacts.command()
+@click.option('--id', 'card_identifier', type=click.STRING, required=False)
+@click.option('--name', 'card_nickname', type=click.STRING, required=False)
+@click.option('--qrcode', is_flag=True, default=None)
+def lookup(card_identifier, card_nickname, qrcode):
+    """"Manage character cards"""
+    emitter = StdoutEmitter()
+    if card_nickname and card_identifier:
+
+        raise click.Abort
+    card = select_card(emitter=emitter, card_identifier=card_identifier or card_nickname)
```

Comment:
should we handle the case when `FileNotFoundError` occurs because of no card dir?

```
  File "/Users/derek/Documents/Github/repos/forks/kieran/nucypher/nucypher/cli/actions/select.py", line 305, in select_card
    for filename in os.listdir(Card.CARD_DIR):
FileNotFoundError: [Errno 2] No such file or directory: '/Users/derek/Library/Application Support/nucypher/cards'
```

Reply:
:heavy_check_mark: